Sources/OuterCore/DominatorTree.swift

```diff
@@ -0,0 +1,82 @@
+/// DominatorTree.swift
+///
+/// Copyright 2018, The Silt Language Project.
+///
+/// This project is released under the MIT license, a copy of which is
+/// available in the repository.
+
+final class DominatorTree {
+  var doms = [Continuation: Continuation]()
```

Comment:
This thing is verbatim from the linked paper and it makes me grumpy. We should look to LLVM’s DominatorTree instead for API design.